### ISSUE TITLE
DataCite titles are not serialized correctly

### ISSUE THREAD

**Issue by bernwaldl** [inveniosoftware/invenio-rdm-records#605]
Package version (if known): 0.29.17

## Describe the bug
When trying out the new DataCite API for InvenioRDM on the DataCite Fabrica Test environment using the demo data on a local installation to automatically create DOIs for documents I get the following error for all entries.
Error message for one record:
```
[2021-05-21 15:52:32,921: WARNING/ForkPoolWorker-7] DataCite provider errored when updating DOI for 10.21979/datacite.tyasc-p2890
[2021-05-21 15:52:32,921: WARNING/ForkPoolWorker-7] Error in title', attribute 'titleType: DOI 10.21979/datacite.tyasc-p2890: [facet 'enumeration'] The value 'Alternativetitle' is not an element of the set {'AlternativeTitle', 'Subtitle', 'TranslatedTitle', 'Other'}. at line 33, column 0
```

## Steps to Reproduce
1. Create and setup own InvenioRDM project
1. Setup DOI creation using DataCite credentials for the DataCite Fabrica test environment
1. Setup services and create demo records using `invenio-cli services setup`
1. Start the instance using `invenio-clil run`
1. Wait until DOI creation is scheduled

## Expected behavior
DOIs are automatically created

## Additional context
According to an RDM partner on discord there is a bug in the DataCite serialization. There seems to be typo somewhere or an error in the "translation" of the titles. I've looked through the source code and from my perspective the error mostly likely lies in the following line.
https://github.com/inveniosoftware/invenio-rdm-records/blob/14baab3db1dc96bf83625e0a78e3774066672e2c/invenio_rdm_records/resources/serializers/datacite/schema.py#L201

**Comment by tmorrell** [comment]
Yup, I think that's the line. I've got some other DataCite serialization issues I'm working on, such as affiliation vs affiliations. My plan:

1. Add test with serializations for all example records
2. Validate all serialized records with datacite schema
3. Fix any issues that come up

**Comment by fenekku** [comment]
fixed in v5.0
